src/Parser/OrientationParser.js

```diff
@@ -0,0 +1,110 @@
+}
+
+// Set East North Up Orientation from geodesic normal
+// target - the quaternion to set
+// up - the normalized geodetic normal to the ellipsoid (given by Coordinates.geodeticNormal)
+var setENUFromGeodesicNormal = (() => {
+    const matrix = new THREE.Matrix4();
+    const elements = matrix.elements;
+    const north = new THREE.Vector3();
+    const east = new THREE.Vector3();
+    return function setENUFromGeodesicNormal(target = new THREE.Quaternion(), up) {
+        // this is an optimized version of matrix.lookAt(up, new THREE.Vector3(0, 0, 0), new THREE.Vector3(0, 0, 1));
+        east.set(-up.y, up.x, 0);
+        east.normalize();
+        north.crossVectors(up, east);
+        north.normalize();
+        elements[0] = east.x; elements[4] = north.x; elements[8] = up.x;
+        elements[1] = east.y; elements[5] = north.y; elements[9] = up.y;
+        elements[2] = east.z; elements[6] = north.z; elements[10] = up.z;
+        return target.setFromRotationMatrix(matrix);
+    };
+})();
+
+/**
+ *
+ * @typedef Attitude
+ * @type {Object}
+ *
+ * @property {Number} omega - angle in degrees
+ * @property {Number} phi - angle in degrees
+ * @property {Number} kappa - angle in degrees
+ * @property {Number} roll - angle in degrees
+ * @property {Number} pitch - angle in degrees
+ * @property {Number} heading - angle in degrees
+ */
+
+/**
+ * @function setFromAttitude
+ * @param {THREE.Quaternion} target Quaternion to set
+ * @param {Attitude} attitude - [Attitude]{@link module:OrientedImageParser~Attitude}
+ * with properties: (omega, phi, kappa), (roll, pitch, heading) or none.
+ * @returns {THREE.Quaternion} target
+ */
+function setFromAttitude(target = new THREE.Quaternion(), attitude) {
+    if ((attitude.roll !== undefined) || (attitude.pitch !== undefined) || (attitude.heading !== undefined)) {
+        return setFromRollPitchHeading(target, attitude.roll, attitude.pitch, attitude.heading);
+    }
+    if ((attitude.omega !== undefined) || (attitude.phi !== undefined) || (attitude.kappa !== undefined)) {
+        return setFromOmegaPhiKappa(target, attitude.omega, attitude.phi, attitude.kappa);
+    }
+    return target.set(0, 0, 0, 1);
+}
+
+export default {
+    /** @module OrientationParser */
+    /** Parse features properties, and add position and quaternion attributes.
+     * @function parse
+     * @param {Object} features - a JSON array of Point Features
+     * @param {Object} options - additional properties.
+     * @param {string} options.crsOut - the CRS of View (must be specified (as 'EPSG:4978') for GlobeView)
+     * @return {Object} a promise that resolves with the features given in parameter
+     */
+    parse(features, options = {}) {
+        const needsENUFromGeodesicNormal = options.crsOut == 'EPSG:4978';
+
+        const ENUQuat = new THREE.Quaternion();
+        for (const feature of features) {
+            const coordinates = feature.vertices[0];
+            feature.position = coordinates.xyz();
```

Comment:
it is not necessary to copy coordinates in `feature.position`

Reply:
ok, let's remove the position and move it in the example code.